src/components/footer/footer-styles.js

```diff
@@ -0,0 +1,208 @@
+import styled from "styled-components"
+import { white, navyBlue, royalBlue } from "../../constants/colors"
+import footerHex from "../../images/footer-hex.svg"
+import { fonts, fontWeights } from "../../constants/typography"
+import { size } from "../../constants/devices"
+
+export const FooterContainer = styled.div`
+  background-color: ${navyBlue};
+  padding: 40px 100px 40px 100px;
+  font-family: "Work Sans", sans-serif;
```

Comment:
I think the font they use for the footer is Futura (either the medium one or Futura book). But it should be the same as the navbar